src/wallet/rpcwallet.cpp

```diff
@@ -1725,7 +1739,7 @@ UniValue listsinceblock(const UniValue& params, bool fHelp)
         CWalletTx tx = (*it).second;
 
         if (depth == -1 || tx.GetDepthInMainChain() < depth)
-            ListTransactions(tx, "*", 0, true, transactions, filter);
+            ListTransactions(tx, "*", 0, true, transactions, filter,"");
```

Comment:
The "pick-all" value for account parameter seems to be *. Why for address, you chose "" instead?